manifests/init.pp

```diff
@@ -40,16 +40,16 @@
   Stdlib::Absolutepath $config_file,
   String[1] $package_name,
   String[1] $service_name,
-  String[1] $module_prefix,
-  Stdlib::Absolutepath $init_config_file,
-  Hash $init_config_hash,
+  String[0] $module_prefix,
```

Comment:
module_prefix is set to `''` on FreeBSD (we do not have modules as packages (yet?)). It is necessary to allow empty String hence the change from:

```diff
-  String[1] $module_prefix,
+  String[0] $module_prefix,
```